File: Modul 05/04/PRAK404-2410817310020-MRizkiDinar.py
out = 0
words = ["penjumlahan","pengurangan","perkalian","pembagian"]

print("Pilih program")
print("1. Penjumlahan")
print("2. Pengurangan")
print("3. Perkalian")
print("4. Pembagian")
print("5. Exit")
ch = input("Masukkan Pilihan : ")
if(ch == "5"):
    print("Terimakasih, telah menggunakan kalkulator MRIZKIDINAR")
    exit()
elif(int(ch) < 1 or int(ch) > 5):
    print("input anda salah, silahkan coba lagi")
    exit()
fst = float(input("Masukkan nilai pertama : "))
scd = float(input("Masukkan nilai kedua : "))

match ch:
    case "1":out = fst + scd
    case "2":out = fst - scd
    case "3":out = fst * scd
    case "4":out = fst / scd
print(f"Hasil {words[int(ch)-1]} antara {fst:.2f} dan {scd:.2f} adalah {out:.2f}")

File: Modul 05/04/PRAK404-2410817310020-MRizkiDinar.c
#include <stdio.h>

int main()
{
    float fst, scd, out = 0.0;
    int ch;
    char *words[] = {"penjumlahan", "pengurangan", "perkalian", "pembagian"};

    printf("Pilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");
    printf("Masukkan Pilihan: ");
    
    scanf("%d", &ch);
    if (ch == 5) {
        printf("Terimakasih, telah menggunakan kalkulator MRIZKIDINAR\n");
        return 0;
    } else if (ch < 1 || ch > 5) {
        printf("Input anda salah, silahkan coba lagi\n");
        return 0;
    }

    printf("Masukkan nilai pertama: ");
    scanf("%f", &fst);
    printf("Masukkan nilai kedua: ");
    scanf("%f", &scd);
    
    switch (ch) {
        case 1: out = fst + scd;break;
        case 2: out = fst - scd;break;
        case 3: out = fst * scd;break;
        case 4: out = fst / scd;break;
    }
    printf("Hasil %s antara %.2f dan %.2f adalah %.2f\n", words[ch - 1], fst, scd, out);
    return 0;
}
